installer/install-clj-kondo-lsp.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
+
+curl -L -o clj-kondo-lsp https://github.com/borkdude/clj-kondo/releases/download/v2020.05.09/clj-kondo-lsp-server-2020.05.09-standalone.jar
```

Comment:
Could you please let new variable `version=2020.05.09` ?

Reply:
Fixed in 1941db0